### ISSUE TITLE
make it into a module

### ISSUE THREAD

**Issue by jongwook** [marl/crepe#7]
- [x] move the content of `crepe.py` into the subdirectory `crepe`
- [x] only expose the public API `crepe.predict` and `crepe.get_activation`
- [x] write `setup.py` so that setuptools can install the module
- [x] include the entry point in the module and in `setup.py` so that the CLI still work
- [x] upload `crepe` to PyPI as version `0.0.x`

**Comment by jongwook** [comment]
PyPI requires the upload size to be <60 MB, but we have ~80 MB. Options are:

1. Retrain the model with slightly lower model capacity multiplier. The current multiplier is 32, and something like 24 or 28 would work
2. Keep the model capacity, but alter the `float32` weights so that it only have about 24 bits of entropy, so that when compressed, the file size is smaller than 60MB.
3. Host the model file somewhere else, like Github, and make the `setup.py` script to download it.
4. Submit the ticket to the warehouse for exception. 

I did reserve the name `crepe` on PyPI.

**Comment by justinsalamon** [comment]
I'm not a fan of (1), because we should aim to release a model that's identical (in architecture/capacity) to the one reported in the paper. (2) might work but I'd be worried about fiddling with the weights post-hoc without knowing whether/how that impacts model performance. (3) would be the cleanest in terms of keeping the model untouched, but might be problematic because we're decoupling the model file from the package version, which might cause problems if we upload updated models in the future. (4) In principle I see no reason not to try this, but I'm guessing it would take a while and there's no guarantee it'll be approved. 

TL;DR - not sure what the best option is here...
